generators/app/utils.js

```diff
@@ -135,12 +133,13 @@ function diffArray(arr1, arr2, generator) {
             if (arr1 === undefined || arr2 === undefined) {
                 return undefined;
             }
-            reduc = _.reduce(arr1, function (result, value, key) {
-                if (arr2[key]) {
-                    return _.isEqual(value, arr2[key]) ? result : result.concat(key);
-                }
-                return false;
-            }, []);
+            // reduc = _.reduce(arr1, (result, value, key) => {
```

Comment:
@antarus @pascalgrimaud Hi guys, just so you know why the linter yelled: there was a declared function here, hence the warning. I admit it's a bit tricky to understand but having this kind of code results in having two (or more, depends on the implem) nested loops:
  - the while
  - the reduce
  - maybe the isEqual (I don't yet know the chosen implem of that one)